tests/plotting/test_plotting.py

```diff
@@ -3003,3 +3003,13 @@ def test_wireframe_color(sphere):
     sphere.plot(
         lighting=False, color='b', style='wireframe', before_close_callback=verify_cache_image
     )
+
+
+@skip_windows
+def test_add_point_scalar_labels_fmt():
+    mesh = examples.load_uniform().slice()
+    p = pyvista.Plotter()
+    p.add_mesh(mesh, scalars="Spatial Point Data", show_edges=True)
+    p.add_point_scalar_labels(mesh, "Spatial Point Data", point_size=20, font_size=36, fmt=r'%.3f')
```

Comment:
I don't think the raw string literal is necessary here. Harmless though.
```suggestion
    p.add_point_scalar_labels(mesh, "Spatial Point Data", point_size=20, font_size=36, fmt='%.3f')
```